lightning/src/ln/channelmonitor.rs

```diff
@@ -1978,9 +1987,7 @@ impl<ChanSigner: ChannelKeys> ChannelMonitor<ChanSigner> {
 			// we call is_resolving_htlc_output here outside of the tx.input.len() == 1 check.
 			self.is_resolving_htlc_output(&tx, height);
 
-			if let Some(spendable_output) = self.is_mine(&tx) {
-				spendable_outputs.push(spendable_output);
-			}
+			self.is_mine(&tx, height);
```

Comment:
is_mine() doesn't really make sense as a name if it has no return value. Maybe detect_spendable_outputs() or so?